releasenotes/notes/manifest-base-cleanup.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: release-notes/v2
+kind: feature
+area: installation
+releaseNotes:
+  - |
+    **Removed** obsolete manifests from the `base` Helm chart. See Upgrade Notes for more information.
+upgradeNotes:
+  - title: "`base` Helm Chart removals"
+    content: |
+      A number of configurations previously present in the the `base` Helm chart were *copied* to the `istiod` chart in a previous releases.
```

Comment:
What exactly is the upgrade implication?   Seems for upgrade, users will still see these, but it is safe for them to manually remove these roles?

Reply:
They are removed on upgrade